### PR TITLE
Use sql "where" clause to list schemas for a vendor

### PR DESCRIPTION
**Pull request by istreeter**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -17,17 +17,17 @@ import sbt._
 object Dependencies {
```
Comment (dilyand) [inline project/Dependencies.scala:17]: Just in case, flagging the TODO in the commit's message.

**Review by dilyand — APPROVED** [review]
Looks great @istreeter ! I commented on a handful of nits. Probably the one I care most about is the naming of the `getSchemasBy*` methods.

File: docker/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+### Set the API key
+
+The easiest way to set an API key is to `exec` into the postgres container:
+
+```bash
+docker-compose exec postgres psql -U postgres igludb -c "$(cat << 'EOH'
+  INSERT INTO iglu_permissions(apikey, vendor, wildcard, schema_action, key_action)
+  VALUES ('5fb4713d-73ad-4163-93a9-2b82f0177c5b', '', 'TRUE', 'CREATE_VENDOR'::schema_action, '{"CREATE", "DELETE"}'::key_action[])
+EOH
```
[the remark this answers is not in the document]
Comment (istreeter) [reply docker/README.md]: I've no idea.  This got removed in a later commit anyway, so if you don't mind I will ignore it 😇

File: config.hocon.sample [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+  password = "iglusecret"
+  driver = "org.postgresql.Driver"
+
+
+  # Configure a connection pool
+  pool = {
+    # type 'hikari' configures a HikariCP connection pool. This is recommended for production deployments.
+    type = "hikari"
+    # maximumPoolSize   = 5        # optional, maximum number of connections in the pool
+    # connectionTimeout = 30000    # optional, millis
```
[the remark this answers is not in the document]
Comment (istreeter) [reply config.hocon.sample]: Yes!  I will change this.

I only left them as seconds to stay compatible with the old config.  But they always annoyed me.

File: src/main/scala/com/snowplowanalytics/iglu/server/storage/Postgres.scala [hunk 4]
```diff
@@ -42,6 +42,22 @@ class Postgres[F[_]](xa: Transactor[F], logger: Logger[F]) extends Storage[F] {
...
+    Stream.eval(logger.debug(s"getSchemasByVendor $vendor")) *>
+      Postgres.Sql.getSchemasByVendor(vendor).stream.transact(xa)
+
+  override def getSchemasByVendorName(vendor: String, name: String)(
+    implicit F: Bracket[F, Throwable]
+  ): Stream[F, Schema] =
+    Stream.eval(logger.debug(s"getSchemasByVendorName $vendor/$name")) *>
+      Postgres.Sql.getSchemasByVendorName(vendor, name).stream.transact(xa)
+
+  override def getSchemasByModel(vendor: String, name: String, model: Int)(
```
[the remark this answers is not in the document]
Comment (istreeter) [reply src/main/scala/com/snowplowanalytics/iglu/server/storage/Postgres.scala:55]: I slightly prefer your final suggestion.  Will change this.

File: config.hocon.sample (hunk 3 above)
[the remark this answers is not in the document]
Comment (istreeter) [reply config.hocon.sample]: See #71

**Pull request merged**